src/function/scalar/operators/add.cpp

```diff
@@ -56,6 +56,20 @@ date_t AddOperator::Operation(int32_t left, date_t right) {
 	return AddOperator::Operation<date_t, int32_t, date_t>(right, left);
 }
 
+template <>
+timestamp_t AddOperator::Operation(date_t left, dtime_t right) {
+	timestamp_t result;
+	if (!Timestamp::TryFromDatetime(left, right, result)) {
+		throw OutOfRangeException("Timestamp out of range");
```

Comment:
Could we add a test that covers this line?

Reply:
Incoming.